metadata-proxy/nginx.conf

```diff
@@ -0,0 +1,47 @@
+user www-data;
+worker_processes 4;
+pid /run/nginx.pid;
+
+events {
+  worker_connections 20;
+}
+
+http {
+  server {
+    listen 127.0.0.1:2020;
+
+    # By default, proxy everything.
+    location / {
+        proxy_pass http://169.254.169.254;
+    }
+
+    # Return 403 for anything too sensitive for pods. Would a regex be better?
```

Comment:
Can we use a whitelist instead of a blacklist approach? My concern is that, for example, when new API versions (or new APIs) come out, suddenly every node has access.